ui/src/app/views/invitation-key/component.css

```diff
@@ -0,0 +1,20 @@
+.input {
+  width: 100%;
+}
+
+.error {
+  background-color: #ffeeee;
+  padding: 0 0.5rem;
+  margin-top: 0.2rem;
+}
+
+.label{
+  width:3rem;
```

Comment:
Nit: spacing.

Reply:
Neha, it looks like you missed a few of my comments. Is that true? If so, could you open a follow-up PR to address them?